addons/slackconnector-1.0.0/docs/overview.md

```diff
@@ -2,7 +2,15 @@
 
 With the use of the token provided during the provisioning of the addon, the Slack Connector allows for sending requests to the Slack Web API. The requests are specified during the installation of the Slack Application to the workspace. The example requests are: posting a message to a specified channel, getting the list of the current users, etc.
 
-## Installation
+#### Prerequisites
+
+- Slack App with desired privileges installed to the destination workspace. See the tutorial provided by Slack on how to setup an application [here](https://api.slack.com/bot-users#getting-started).
+
+    After creating the app:
+    - The Slack Application's {SIGNING_SECRET} is used for validating requests coming from Slack by verifying its unique signature. Find it in the [Application's](https://api.slack.com/apps) **Settings** in the **Basic Information** section. [Learn more](https://api.slack.com/docs/verifying-requests-from-slack).
+    - The Slack Application's {SLACK_TOKEN} is used for requests authorization. Find it in the [Application's](https://api.slack.com/apps) **Features** in the **OAuth & Permissions** section. Depending on the usecase it will be *OAuth Access Token* or *Bot User OAuth Access Token*. [Learn more](https://api.slack.com/docs/oauth).
```

Comment:
```suggestion
    - The Slack Application's {SLACK_TOKEN} is used for requests authorization. Find it in the [Application's](https://api.slack.com/apps) **Features** in the **OAuth & Permissions** section. Depending on the usecase it will be **OAuth Access Token** or **Bot User OAuth Access Token**. [Learn more](https://api.slack.com/docs/oauth).
```